charts/payment-jobs/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
we don't normally check this in to make it easier to update and to allow semver range to get picked up